src/utils/localStorage.ts

```diff
@@ -0,0 +1,8 @@
+export const getSearchParam = (key: string) => {
+  const searchParam = localStorage.getItem(key) || '';
+  return searchParam;
+};
+
+export const setSearchParam = (key: string, value: string) => {
+  localStorage.setItem(key, value);
+};
```

Comment:
I hardly understand why you store seachParamters in local storage. Is this just for passing data in between your components?

If so, I think you better to use Context or composition

Reply:
In our requirements, it was necessary store the input value in local storage, but I had already used the useContext for this in new branch😊

Reply:
cool, overally, your PR looks good, for the current stage

Reply:
thanks, but I really want to rewrite everything)